scenes/wifi_marauder_scene_start.c

```diff
@@ -33,6 +33,13 @@ const WifiMarauderItem items[NUM_MENU_ITEMS] = {
      NO_ARGS,
      FOCUS_CONSOLE_END,
      SHOW_STOPSCAN_TIP},
+    {"Led",
+     {"HEX", "Pattern"},
```

Comment:
Can you change "Led" to "LED" and make the parameters all lowercase? I also think it should be moved under Signal Monitor in the menu (or somewhere around there).

Note: on my wifi dev board, this gives me an error message "hardware does not support neopixel". It works on the pro.

Reply:
Fixed the formatting. The official wifi dev board does not have a neo pixel. It has some other type of RGB LED and I have to write some code to integrated that hardware into the command